charts/ditto/Chart.yaml

```diff
@@ -17,7 +17,7 @@ description: |
   A digital twin is a virtual, cloud based, representation of his real world counterpart
   (real world “Things”, e.g. devices like sensors, smart heating, connected cars, smart grids, EV charging stations etc).
 type: application
-version: 3.2.1
+version: 3.3.0
```

Comment:
Please adjust the "version" to 3.2.2 - we try to keep at least major+minor version in sync with the appVersion.

And while you are at it: could you please also adjust the `appVersion to 3.2.1`? 
We did a Ditto release 3.2.1 but did not yet adjust the chart.

Reply:
Sorry, I edited the minor thinking adding an endpoint was a feature. I immediately correct the version and also the appVersion as indicated